docs/tutorials/basic/symbol.md

```diff
@@ -56,60 +111,150 @@ net = mx.sym.SoftmaxOutput(data=net, name='out')
 mx.viz.plot_network(net, shape={'data':(100,200)})
 ```
 
-### Modulelized Construction for Deep Networks
+Each symbol takes a (unique) string name.
+*Variable* typically represents inputs to the computation graph.
+Symbols defined via operators typically take other symbols as their input data,
```

Comment:
same as above. The concepts are confused here. Symbols corresponds to NDArray. They both represent a single tensor. Operators represent the computation between tensors. Operators take symbol (or NDArray) as inputs and produce it as output.